modules/react/button/lib/ExternalHyperlink.tsx

```diff
@@ -10,7 +10,7 @@ export interface ExternalHyperlinkProps extends HyperlinkProps {
    * Informs a screen reader user the link will open in a new window. It is read after the link text.
    * This value will need to be translated.
    */
-  iconLabel: string;
+  iconLabel?: string;
```

Comment:
Question: If this is required for users to add, why are we making the prop optional? Is that what we're doing on other components?

Reply:
I think we've talked about this but I don't want to force people to add aria-labels. This will break consumers and we can't codemod it so a "softer landing" is to make it optional but ensure we communicate that for proper accessibility, they must provide a label so all of our examples are updated.